tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -633,3 +642,184 @@ def custom_continue_workflow(self, *args, **kwargs):
     assert obj.extra_data['is-update']
     assert obj.extra_data['approved']
     assert obj.status == ObjectStatus.COMPLETED
+
+
+def test_validation_error_callback_with_a_valid(workflow_app):
+    valid_record = {
+        '_collections': [
+            'Literature',
+        ],
+        'document_type': [
+            'article',
+        ],
+        'titles': [
+            {'title': 'A title'},
+        ],
+    }
+
+    eng_uuid = start('article', [valid_record])
+
+    eng = WorkflowEngine.from_uuid(eng_uuid)
+    obj = eng.objects[0]
+
+    assert obj.status != ObjectStatus.ERROR
+
+    response = do_validation_callback(
+        workflow_app,
+        obj.id,
+        obj.data,
+        obj.extra_data
+    )
+
+    expected_error_code = 'WORKFLOW_NOT_IN_ERROR_STATE'
+    data = json.loads(response.get_data())
+
+    assert response.status_code == 400
+    assert expected_error_code == data['error_code']
+
+
+def test_validation_error_callback_with_validation_error(workflow_app):
+    invalid_record = {
+        '_collections': [
+            'Literature',
+        ],
+        'document_type': [
+            'article',
+        ],
+        'titles': [
+            {'title': 'A title'},
+        ],
+        'preprint_date': 'Jessica Jones'
+    }
+
+    obj = workflow_object_class.create(
+        data=invalid_record,
+        data_type='hep',
+        id_user=1,
+    )
+    obj_id = obj.id
+
+    with pytest.raises(ValidationError):
+        start('article', invalid_record, obj_id)
```

Comment:
Nit: maybe assert that the workflow object is in ERROR state after this? Just to be 100% clear.